src/StaticHashing.hpp

```diff
@@ -3,22 +3,33 @@
 
 #include "DiskManager.hpp"
 #include "HashingScheme.hpp"
+#include "Bucket.hpp"
 
 template<typename K, typename V>
 class StaticHashing : HashingScheme<K, V> {
 public:
     bool insert(const K &key, const V &value) override {
-        return false;
+      std::size_t hashed_key = std::hash(key);
+      return Bucket.insert(hashed_key%num_slots,value)
```

Comment:
Overflow chaining??

Reply:
How did this even compile?? you have to instantiate a Bucket object before calling insert on it.